tests/backup/backup_sse_test.go

```diff
@@ -255,7 +255,8 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
 					go func(scName string) {
 						defer GinkgoRecover()
 						defer wg.Done()
-						err = CreateRestore(restoreName, backupsWithSse[0], namespaceMap, SourceClusterName, orgID, ctx, storageClassMapping)
+						appContextsToBackup = FilterAppContextsByNamespace(scheduledAppContexts, []string{bkpNamespaces[midpoint]})
```

Comment:
This might fail because we are doing validation in parallel.

Reply:
Corrected in https://github.com/portworx/torpedo/pull/1968